morse_decoder.rb

```diff
@@ -0,0 +1,53 @@
+MORSE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z',
+  '-----' => '0',
+  '.----' => '1',
+  '..---' => '2',
+  '...--' => '3',
+  '....-' => '4',
+  '.....' => '5',
+  '-....' => '6',
+  '--...' => '7',
+  '---..' => '8',
+  '----.' => '9'
+}.freeze
+
+def decode_char(random_char)
+  puts MORSE[random_char]
+end
+
+def decode_word(random_char)
+  random_char.split.map { |string| MORSE[string] }.join
+end
+
+def decode_sentence(random_char)
+  words = random_char.split('   ')
+  puts words.map { |word| decodeWord word }.join(' ')
+end
```

Comment:
- [ ] Please your `decode_sentence` method is not working since it is throwing an error that you are calling an undefined method `decodeWord`. I guess you are trying to call the defined method `decode_word` but you made a typo :smile: 
  
  ![image](https://user-images.githubusercontent.com/92303891/232047685-7a0512ce-65cc-4fc7-a724-ea5eae8b2bd1.png)